src/style.css

```
@@ -0,0 +1,38 @@
+body {
+  background-color: #000;
+  margin: 20px;
+}
+
+.yellowblock {
+  background-color: #ffa500;
+  border: 1px solid #e95d49;
+  padding: 10px;
+}
+
+.blueblock {
+  padding: 10px ;
```

Не залишай пусті місця
```suggestion
  padding: 10px;
```